gradio/blocks.py

```diff
@@ -1637,11 +1637,11 @@ def queue(
             with gr.Blocks() as demo:
                 button = gr.Button(label="Generate Image")
                 button.click(fn=image_generator, inputs=gr.Textbox(), outputs=gr.Image())
-            demo.queue(concurrency_count=3)
+            demo.queue(concurrency_count=1)
```

Comment:
Since `concurrency_count=1` by default, it seems redundant to include it as a parameter. How about this instead?
```suggestion
            demo.queue(max_size=10)
```

Reply:
yes, I was not sure about not mentioning concurrency_count at all. This param can still be cool if there is no GPU usage, for example, a chatbot that is calling an API.

Reply:
I think including it is ok to let users know that it exists. But don't feel super strongly since it's in the docstring.

Reply:
Yup sounds good either way